forge/logging/2023/index.js

```diff
@@ -0,0 +1,114 @@
+const PROJECT_ID = "pyret-examples";
```

Comment:
This cloud function does work for Forge and Examplar.

It would be AMAZING to have a new cloud function that works for Forge alone. But it's got to allow public access. How to do that?